services/ChartJsUtils.cfc

```diff
@@ -123,6 +124,12 @@ component {
 			config.options.scales[ scale.id ] = scaleConfig;
 		} );
 
+		// Plugins
+		var plugins = "";
+		chart.getPlugins().each( function( plugin, index ) {
```

Comment:
Is this not just: 

```cfc
var plugins = ArrayToList( chart.getPlugins() );
```

?

Or, evn more concise:

```cfc
config.plugins = [ ArrayToList( chart.getPlugins() ) ];
```

The placeholder, etc. is confusing to me - why do we need to serialize first and then replace the placeholder string? (line 171 below)

Reply:
I've also added the usage in README docs